bin/cli

```diff
@@ -222,7 +222,9 @@ function warnIfNpmStart (argv, env) {
           '--no-audit',
           '--loglevel', 'error',
           kitDependency,
-          'govuk-frontend'
+          'govuk-frontend',
+          // older versions of kit came with GOV.UK Notify support, keep it
+          'notifications-node-client@5'
```

Comment:
does this only keep it if it is version 5? Wondering if we just keep it whatever it is

Reply:
The migration script removes the whole package.json and replaces it with a new one. This line just tells npm to install the latest version 5 release of the package, v5 being the version the kit v12 release comes with. I thought that we probably didn't want to install any future hypothetical breaking release of the notify client, because any prototype that is being migrated will come with v5.

Reply:
ah I see thanks